src/services/section-helper.js

```diff
@@ -110,3 +112,49 @@ export function extractCdnResources(sections, pluginInfoFactory) {
     }, new Set())
   ].sort();
 }
+
+function _clipboardEncode(clipboardObject, encryptionKey) {
+  const jsonText = JSON.stringify(clipboardObject || null);
+  return AES.encrypt(jsonText, encryptionKey);
+}
+
+function _clipboardDecode(clipboardText, encryptionKey) {
+  try {
+    const decryptedText = AES.decrypt(clipboardText, encryptionKey).toString(cryptoJsUtf8Encoder);
+    return JSON.parse(decryptedText);
+  } catch {
+    return null;
+  }
+}
+
+function _isValidClipboardObject(clipboardObject) {
```

Comment:
Tiny thing, since this is specific for sections,  something like `_isValidClipboardSection` might be better.

Reply:
Sure, the "object" part is already in the param name anyway. Done.